astropy/coordinates/erfa_astrom.py

```diff
@@ -28,6 +30,25 @@
 __all__ = []
 
 
+class _ReferenceCoefficients(NamedTuple):
```

Comment:
`pyerfa` could define this so we would't have to.

Reply:
Agreed that this makes more sense in `pyerfa`. Now have gone beyond raising an issue: https://github.com/liberfa/pyerfa/pull/176